Adyen/Core/Core Protocols/AnyDropInComponent.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Any DropIn Component.
+public protocol AnyDropInComponent: PresentableComponent {
```

Comment:
Just a thought: maybe we can come up with bit more generic name?
Like "payment interface" or "checkout interface" ?